File: backend/ruby_foundations/exercises/small_problems_101/easy_7/multiply_lists.rb
def multiply_list(array1, array2)
  results = []
  array1.each_index do |index|
    results << array1[index] * array2[index]
  end
  results
end

puts multiply_list([3, 5, 7], [9, 10, 11]) == [27, 50, 77]

File: frontend/javascript/exercises/easy_4/multiply_lists.js
function multiplyList(array1, array2) {
  var result = []
  for (var i = 0; i < array1.length; i++) {
    result.push(array1[i] * array2[i]);
  }
  return result;
}

console.log(multiplyList([3, 5, 7], [9, 10, 11])); // [27, 50, 77]